.github/workflows/conformance.yml

```diff
@@ -319,10 +370,6 @@ jobs:
       run: docker-compose -f ./.github/infrastructure/docker-compose-rabbitmq.yml -p rabbitmq up -d
       if: contains(matrix.component, 'rabbitmq')
 
-    - name: Start aws snssqs
```

Comment:
Should we keep it too? It can be helpful to run the tests for PRs and not wait for the code to be merged.